### ISSUE TITLE
extended testing failure on aix

### ISSUE THREAD

**Issue by jdekonin** [eclipse/openj9#1355]
What will be a daily pipeline job is failing to find/load a test library.

https://ci.eclipse.org/openj9/job/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/1/consoleFull
```
===============================================
Running test memoryCategories_0 ...
===============================================
test with NoOptions
{ mkdir -p "/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/test/TestConfig/scripts/testKitGen/../../../TestConfig/test_output_152030461914/memoryCategories_0"; \
        cd "/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/test/TestConfig/scripts/testKitGen/../../../TestConfig/test_output_152030461914/memoryCategories_0"; \
        "/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/build/aix-ppc64-normal-server-release/images/j2sdk-image/jre/bin/java"  -Xcompressedrefs  \
-cp "/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/test/TestConfig/scripts/testKitGen/../../../../jvmtest/TestConfig/resources:/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/test/TestConfig/scripts/testKitGen/../../../../jvmtest/TestConfig/lib/testng.jar:/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/test/TestConfig/scripts/testKitGen/../../../../jvmtest/TestConfig/lib/jcommander.jar:/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/test/TestConfig/scripts/testKitGen/../../../../jvmtest/Java8andUp/GeneralTest.jar" \
org.testng.TestNG -d "/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/test/TestConfig/scripts/testKitGen/../../../TestConfig/test_output_152030461914/memoryCategories_0" "/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/test/TestConfig/scripts/testKitGen/../../../../jvmtest/Java8andUp/testng.xml" \
-testnames memoryCategories \
-groups level.extended \
-excludegroups d.*.aix_ppc-64_cmprssptrs,d.*.arch.ppc,d.*.os.aix,d.*.bits.64,d.*.generic-all; \
if [ $? -eq 0 ] ; then echo "memoryCategories_0""_PASSED"; else echo -e "\nmemoryCategories_0""_FAILED\n"; fi; } 2>&1 | tee -a "/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/test/TestConfig/scripts/testKitGen/../../../TestConfig/test_output_152030461914/TestTargetResult"
[IncludeExcludeTestAnnotationTransformer] [INFO] exclude file is /home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/test/TestConfig/../../jvmtest/TestConfig/resources/excludes/latest_exclude_SE90.txt
...
... TestNG 6.10 by C?dric Beust (cedric@beust.com)
...

[TestNG] Running:
  /home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/openj9/jvmtest/Java8andUp/testng.xml

FAILED CONFIGURATION: @BeforeMethod setUp
java.lang.AssertionError: j9ben (Not found in java.library.path)
```

It would appear, if library path is LIBPATH, is set to 
```
library path = /home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/build/aix-ppc64-normal-server-release/images/j2sdk-image/jre/lib/ppc64/compressedrefs:/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/build/aix-ppc64-normal-server-release/images/j2sdk-image/jre/lib/ppc64:/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/build/aix-ppc64-normal-server-release/images/j2sdk-image/jre/lib/ppc64/j9vm:/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/build/aix-ppc64-normal-server-release/images/j2sdk-image/jre/lib/ppc64:/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/build/aix-ppc64-normal-server-release/images/j2sdk-image/jre/lib/ppc64/jli:/home/u0020236/workspace/Test-Extended-JDK8-aix_ppc-64_cmprssptrs/build/aix-ppc64-normal-server-release/images/j2sdk-image/jre/../lib/ppc64:/usr/lib64:/usr/lib
```
libj9ben.so can be found in : 
/home/u0020236/workspace/PullRequest-Sanity-JDK8-aix_ppc-64_cmprssptrs-OpenJ9/build/aix-ppc64-normal-server-release/images/j2sdk-image/jre/lib/ppc64/compressedrefs/

fyi @llxia

**Comment by llxia** [comment]
From the test output, it seems we need to export LIBPATH in test.

**Comment by smlambert** [comment]
Either this needs a fix this asap or disable the test for AIX platform as AIX has been added to the PR testing.  

Likely disabling the test for AIX should have happened as part of the PR that added AIX to PR testing, and raise this issue, if we are keeping to the always to run green philosophy.

**Comment by pshipton** [comment]
@jdekonin please create a PR to exclude the test.

**Comment by llxia** [comment]
I should have a fix today. If not, I will exclude the test.

**Comment by smlambert** [comment]
Thanks @llxia !

**Comment by llxia** [comment]
Test is excluded on AIX.

I will deliver #1365 (fix LIBPATH) until #1377 is fixed.
